### PR TITLE
Join section elements/layout

### PR DESCRIPTION
**Pull request by tdoug**
I'm not sure if you designed this layout, but I had some questions:

1) Shouldn't the "call to action" text above the Gitter icon also be centered?
2) Can we part out some of margins of buttons so they're properly centered without overriding and adding padding and sizes for each one?
3) Shouldn't the partnership section, and all other sections, be vertically and horizontally centered?  Is the grid offset for a reason or should I fix these?
4) Do you care if I just finish the entire layout and keep it somewhat sane/standardized?  How buck wild can I go?

### CODE REVIEW

**Comment by bahtou** [comment]
**1, 3, 4**
There is a constaint grid system for both mobile & web. The README has a brief summary on the grid harmonics. Basically, I drew inspiration from Andy Clarke. There are many design architects that have their own methodologies, and Andy Clarke stands out to me as a guide in the right direction because he bridges the art, architecture, and css in web developer terminology. Really recommend the following:

https://www.youtube.com/watch?v=eUeoLUjOUHw
https://stuffandnonsense.co.uk/artdirectionfortheweb/
https://every-layout.dev/ (not Andy Clarke but good resource)

In the sketch file (if you have sketch?) there are files labeled `1-1-3-1-2-2-1-3-1-1`, `1-1-2-4-4-2-1-1`, `2-1-1-2-2-1-1-2`. If you toggle these you see the grid layout and the content lining up accordingly.

That being said, the look & feel of the site has been through the gauntlet more times than it should (approximately 5 months worth of helpful criticisms, opinions, etc, before it was announced at the Dec. meetup -- sad. I know). The layout is sane/standardized. I noticed that some of the CSS needs to be updated to accomodate the responsiveness but the grid should serve as a guide.

**2**
ah. meant to fix that button component.
Right now it's an `<input />` but swapping that out to `<button type="submit">`.
Removing the `margin-top` as well. Pushing an update to master.
Let me know if this resolves any issues with it.

File: src/sections/Join/index.js [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+  }
+
+  return (
+    <section id="join" className={joinLayout}>
+      <div className={formBlock}>
+        <h1 className={formTitle}>Join</h1>
+        <span className={formSubtitle}>Subscribe to monthly meetup events & get notified of upcoming events like community workshops, hackathons, presentations and recordings.</span>
+
+        <form id="joinForm" className={joinForm} onSubmit={handleSubmit}>
+          <fieldset className={inputFields}>
```
Comment (bahtou) [inline src/sections/Join/index.js]: we can get rid of the single fieldset here. only one element to group doesn't make much sense

File: src/sections/Join/index.js [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
+            <input id='joinEmail' type="text" value={email} onChange={ev => setEmail(ev.target.value)} required />
+          </fieldset>
+          <FormButton className={formButton}>Subscribe</FormButton>
```
Comment (bahtou) [inline src/sections/Join/index.js]: pull master and let me know if the `FormSubmitButton` works

File: src/sections/Join/index.js (hunk 1 above)
Comment (bahtou) [reply src/sections/Join/index.js]: also, `inputFields` doesn't appear to be used

File: src/sections/Join/index.js (hunk 2 above)
Comment (tdoug) [reply src/sections/Join/index.js]: Changed this up, and yes the new button component worked and required less changes.

I think this PR is ready.  Let me know if you need any more help with your meetup or anything else.

**Comment by bahtou** [comment]
looks good :+1:

**Pull request merged**